test/lib/rails-jquery-autocomplete_test.rb

```diff
@@ -15,13 +15,14 @@ class ::Movie ; end
       end
 
       should 'respond to the action' do
-        assert_respond_to @controller, :autocomplete_movie_name
+        assert_respond_to @controller, :autocomplete_name
       end
 
       should 'render the JSON items' do
-        mock(@controller).get_autocomplete_items({
-          :model => Movie, :method => :name, :options => @options, :term => "query"
-        }) { @items }
+        mock(@controller).get_autocomplete_items(
+          Movie,
+          { :method => :name, :options => @options, :term => "query" }
```

Comment:
Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.